doc/manual/50-feature-details.en.md

```diff
@@ -73,6 +70,52 @@ incremental backup as follows:
 barman backup --reuse-backup=link <server_name>
 ```
 
+> **NOTE:** Unlike Postgres block-level incremental backups, Rsync file-level incremental backups are independent
+> on their own, meaning that a backup that reused a previous backup for deduplication is not compromised
+> in any way if that backup is deleted. As mentioned, deduplication in Rsync backups is implemented with the use
+> of hard links, so when a previously reused backup is deleted, files shared with other backups will
+> still remain on disk, only being removed when the last backup using those files is also deleted. It also
+> means that there’s no need to take new full backups when a previous full backup is deleted, each backup
+> will still have all files and can be reused without any concerns.
```

Comment:
```suggestion
> still remain on disk, only being removed when the last backup using those files is also deleted. It also
> means that there’s no need to take full backups when a previous full backup is deleted, each backup
> will still have all files and can be reused without any concerns.
> Along the same lines, there is no need to ever take a full backup with rsync by using `reuse_backup = off`. If it is the first backup being taken with `reuse_backup = link`, in essence
> it behaves like `off` because there are no existing files to create hard-links on.
```